docs/ci/popperyml.md

```diff
@@ -67,15 +93,44 @@ popper init mypipe --env=alpine-3.4
 The above specifies that the pipeline named `mypipe` will be executed 
 inside a docker container using the `alpine-3.4` popper check image.
 
+## Metadata
+
+The `metadata` YAML entry specifies the set of data that gives information
+about the user's project. 
+
+It can be added using the `popper metadata --add command`
```

Comment:
this fragment should be part of the previous sentence, and followed by "For example" (all in one single paragraph):

> The `metadata` YAML entry specifies the set of data that gives information about the user's project.  It can be added using the `popper metadata --add command`. For example: